src/Flysystem/FlysystemAssetStore.php

```diff
@@ -579,6 +579,7 @@ function (ParsedFileID $parsedFileID, Filesystem $fs, FileResolutionStrategy $st
                 $destParsedFileID = $parsedFileID->setFilename($newName);
 
                 // Move all variants around
+                $tryTruncate = [];
```

Comment:
```suggestion
                $dirsToTruncate = [];
```
Repeat for all other examples below

Reply:
Done